src/routes/AuthRedirect/AuthRedirect.test.tsx

```diff
@@ -0,0 +1,54 @@
+import {API} from "api";
+
+import {render} from "@testing-library/react";
+import {createMemoryHistory} from "history";
+import {Router} from "react-router";
+import AuthRedirect from "./AuthRedirect";
+
+jest.mock("api", () => {
+  return {
+    API: {
+      verifyGoogleSignIn: jest.fn(),
+    },
+  };
+});
+
+beforeEach(() => {
+  (API.verifyGoogleSignIn as jest.Mock).mockClear();
+});
+
+describe("AuthRedirect", () => {
+  test("should render loading screen while waiting for server", () => {
```

Comment:
This test sounds like you wanted to check the snapshot to contain the loading screen